pkg/cloud/onprem/onprem.go

```diff
@@ -16,12 +16,15 @@ package onprem
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
 	"go.starlark.net/starlark"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 
 	"github.com/cruise-automation/isopod/pkg/cloud"
+	"github.robot.car/cruise/isopod/pkg/vault"
```

Comment:
import prefix should be `github.com/cruise-automation/isopod/`

Reply:
We caught this at the same time. Will reopen